src/ace/models.py

```diff
@@ -460,3 +468,30 @@ class Var(Base, AdmObjMixin, ParamMixin, TypeUseMixin):
     ''' The initial value as text ARI '''
     init_ari = Column(PickleType)
     ''' Resolved and decoded ARI for ivar:`init_value`. '''
+
+class Sbr(Base, AdmObjMixin):
+    ''' State Based Rule '''
+    __tablename__ = "sbr"
+    # Unique ID of the row
+    id = Column(Integer, primary_key=True)
+    # ID of the file from which this came
+    module_id = Column(Integer, ForeignKey("adm_module.id"))
+    # Relationship to the :class:`AdmModule`
+    module = relationship("AdmModule", back_populates="sbr")
+
+    action = Column(PickleType)
```

Comment:
I recommend to steal the way other tables handle ARIs (e.g. the `var` initial value) which is to retain the original text form and the decoded form.